src/pages/Buttons/bottom/ScrollToBottomButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
```

Comment:
You can try to write this code and file add in components folder and remove from pages folder for **Good Practice**